main.py

```diff
@@ -1,10 +1,61 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_data_file:
+        dict_players = json.load(players_data_file)
+
+    for player in dict_players:
+        race_data = dict_players[player]["race"]
+        if Race.objects.filter(
+                name=race_data["name"]
+        ).exists() is False:
+            race = Race.objects.create(
+                name=race_data["name"],
+                description=race_data["description"]
+            )
+        else:
+            race = Race.objects.get(
+                name=race_data["name"]
+            )
```

Comment:
Use get_or_create
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create
The same with other places